plugins/org.locationtech.udig.catalog.db2/src/org/locationtech/udig/catalog/internal/db2/ui/DB2WizardPage.java

```diff
@@ -48,11 +47,12 @@
  * </p>
  * 
  * @author Justin Deoliveira, jdeolive, for Refractions Research, Inc.
- * @author dadler
  * @author Jesse Eichar, jeichar, for Refractions Research, Inc.
  * @author Richard Gould, rgould, for Refractions Research, Inc.
  * @author Adrian Custer, acuster.
  * @since 1.0.1
+ * @author David Adler, dadler, for AdtechGeospatial 
+ * @since 2.0.x 
```

Comment:
When and what is the next release to go out?  I would like to get these fairly minor fixes in as soon as possible.
Should I change the prologs and commit again?